config/database.example.yml

```diff
@@ -5,16 +5,17 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: mysql2
-  encoding: utf8
+  adapter: postgresql
+  encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  username: root
-  password: root
-  socket: /tmp/mysql.sock
+  username: postgres
+  password: postgres
```

Comment:
Most of the times the password for the postgres user will be empty, Maybe we can replace this with `''`

Reply:
I think the default is postgres . we can add a note in readme to ask the user to check with ''

Reply:
It says here that [here](https://www.liquidweb.com/kb/what-is-the-default-password-for-postgresql/) isn't a default password initially. Maybe mentioning it as `password: <password>` along with a note in the readme about filling in the field can work. :slightly_smiling_face:

Reply:
I think it differs with operating systems :) Not sure though
password: <password> along with a note  is the best option